assignments/prototypes.js

```diff
@@ -41,7 +41,41 @@
 
 // Test you work by uncommenting these 3 objects and the list of console logs below:
 
-/*
+
+  function GameObject(gameObjectAttributes){
+    this.createdAt = gameObjectAttributes.createdAt;
+    this.dimensions = gameObjectAttributes.dimensions;
```

Comment:
This function is a good candidate for [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring):
```javascript
function GameObject({ createdAt, dimensions }) {
  this.createdAt = createdAt;
  this.dimensions = dimensions;
}
```

Reply:
for the sake of brevity.